src/game_interpreter_map.cpp

```diff
@@ -712,15 +713,63 @@ bool Game_Interpreter_Map::CommandPlayMovie(lcf::rpg::EventCommand const& com) {
 	return true;
 }
 
-bool Game_Interpreter_Map::CommandOpenSaveMenu(lcf::rpg::EventCommand const& /* com */) { // code 11910
+bool Game_Interpreter_Map::CommandOpenSaveMenu(lcf::rpg::EventCommand const& com) { // code 11910
 	auto& frame = GetFrame();
 	auto& index = frame.current_command;
 
 	if (Game_Message::IsMessageActive()) {
 		return false;
 	}
 
-	Scene::instance->SetRequestedScene(std::make_shared<Scene_Save>());
+	const int current_system_function = com.parameters[0];
+
+	// Handle save menu (default behavior)
+	if (!Player::IsPatchManiac() || current_system_function <= 0) {
+		Scene::instance->SetRequestedScene(std::make_shared<Scene_Save>());
+		++index;
+		return false;
+	}
+
+	// Parse common parameters
+	const int fullscreen_mode = com.parameters[1]; // Broken in Maniac.
+	const int pause_while_debugging = com.parameters[1]; // unused in our ingame debug screen.
+
+	const int actor_index = ValueOrVariable(com.parameters[1], com.parameters[2]);
+	const bool is_db_actor = ValueOrVariable(com.parameters[3], com.parameters[4]);
+
+	switch (current_system_function) {
+	case 1: // Load menu
+		return CommandOpenLoadMenu(com);
+	case 2: // Game menu
+		Scene::instance->SetRequestedScene(std::make_shared<Scene_Menu>());
+		break;
+	case 3: // Toggle fullscreen
+		// TODO? Implement fullscreen mode once maniacs supports it
+		return CommandToggleFullscreen(com);
+	case 4: // Settings menu
+		return CommandOpenVideoOptions(com);
+	case 5: // Debug menu
+		Scene::instance->SetRequestedScene(std::make_shared<Scene_Debug>());
+		break;
+	case 6: // License information menu
+		// TODO? Implement license information menu
+		return true;
+	case 7: // Reset game
+		return CommandReturnToTitleScreen(com);
+	case 8: // EASYRPG Inventory menu
```

Comment:
Is it intentional that an enabled direct menu patch can redirect these menu calls? (the dmpatch redirection is in ``RequestMainMenuScene``)